src/Render.cc

```diff
@@ -14,9 +14,9 @@ using namespace snowcrash;
 
 namespace drafter {
 
-    RenderFormat findRenderFormat(std::string contentType) {
+    RenderFormat findRenderFormat(std::string& contentType) {
 
-        if (contentType == "application/json") {
+        if (contentType.find("application/json") != std::string::npos) {
```

Comment:
since you have got into this I would consider supporting - JSON-based formats (e.g. `application/hal+json` or `application/vnd.github.v3+json` etc.)